cfme/common/vm.py

```diff
@@ -446,33 +446,69 @@ def _refresh():
             self.load_details()
 
     def set_ownership(self, user=None, group=None, click_cancel=False, click_reset=False):
-        """Set ownership of the VM/Instance or Template/Image"""
-        view = navigate_to(self, "SetOwnership")
-        view.form.fill({'user_name': user, 'group_name': group})
+        """Set instance ownership
+
+        Args:
+            user (str): username for ownership
+            group (str): groupname for ownership
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
+        """
+        view = navigate_to(self, 'SetOwnership')
+        fill_result = view.form.fill({
+            'user_name': user,
+            'group_name': group})
+        if not fill_result:
+            view.flash.assert_no_error()
```

Comment:
**QUESTION**
I'm not sure why you do assert here after filling the form.

Reply:
@quarckster , I overlooked that one :).

Fixed it now.